src/ImageService.Data/ImageMessageRepository.cs

```diff
@@ -1,77 +1,64 @@
-﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
-using LT.DigitalOffice.ImageService.Data.Provider;
-using LT.DigitalOffice.ImageService.Models.Db;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.ImageService.Data.Provider;
+using LT.DigitalOffice.ImageService.Models.Db;
+using Microsoft.Data.SqlClient;
 
 namespace LT.DigitalOffice.ImageService.Data
 {
-    public class ImageMessageRepository : IImageMessageRepository
+  public class ImageMessageRepository : IImageMessageRepository
+  {
+    private readonly IDataProvider _provider;
+    public ImageMessageRepository(IDataProvider provider)
     {
-        private readonly IDataProvider _provider;
-        public ImageMessageRepository(IDataProvider provider)
-        {
-            _provider = provider;
-        }
-
-        public List<Guid> Create(List<DbImageMessage> imagesMessages)
-        {
-            if (imagesMessages.Contains(null))
-            {
-                return null;
-            }
-
-            _provider.ImagesMessages.AddRange(imagesMessages);
-            _provider.Save();
+      _provider = provider;
+    }
 
-            return imagesMessages.Select(x => x.Id).ToList();
-        }
+    public List<Guid> Create(List<DbImageMessage> imagesMessages)
+    {
+      if (imagesMessages.Contains(null))
```

Comment:
why if any image is null then we dont create all?

Reply:
done